VERSION

```diff
@@ -1 +1 @@
-1.5.1
+1.6.0
```

Comment:
This is going to be updated by the bot (e.g. for 1.5.0: https://github.com/Kong/gateway-operator/pull/1333/files).

https://github.com/Kong/gateway-operator/blob/a757bdbb4f4b3eb2560deb408ec02f7745c2a3dd/.github/workflows/__release-workflow.yaml#L310-L333

Reply:
Was this mentioned in the release check list to change these files? It shouldn't 🤔

Reply:
The release checklist mentioned that CI will automatically update it, but its order may need to be adjusted. I will create a PR to adjust it.